app/src/main/java/io/gnosis/safe/ui/safe/add/AddSafeNameFragment.kt

```diff
@@ -51,7 +54,14 @@ class AddSafeNameFragment : BaseFragment<FragmentAddSafeNameBinding>() {
                 is AddSafeNameState -> {
                     state.viewAction?.let { action ->
                         when (action) {
-                            is BaseStateViewModel.ViewAction.CloseScreen -> findNavController().navigateUp()
+                            is BaseStateViewModel.ViewAction.CloseScreen -> {
+                                (requireContext().getSystemService(Context.INPUT_METHOD_SERVICE) as? InputMethodManager?)
```

Comment:
this could be extracted to a private method or placed in utils as extension function

Reply:
Didn't we have utils in the old app for this?

Reply:
In svalinn in fact. I will replace it with that.